app/src/main/java/com/github/fribourgsdp/radio/data/view/UserProfileActivity.kt

```diff
@@ -47,17 +49,21 @@ open class UserProfileActivity : MyAppCompatActivity(), KeepOrDismissPlaylistDia
     private lateinit var userIcon : ImageView
     private var signedIn : Boolean = false
 
+
     private var db = initializeDatabase()
 
+    protected lateinit var googleSignIn : GoogleSignIn
 
     //firebase auth
     private lateinit var firebaseAuth: FirebaseAuth
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_user_profile)
+        googleSignIn = GoogleSignIn(this, getString(R.string.default_web_client_id))
         instantiateViews()
 
+
         val fromGoogle = intent.getBooleanExtra("FromGoogle",false)
```

Comment:
You can delete this line as it the fromGoogle value is not more used.

Reply:
It was done between the time you read the code and wrote your comment. 
If you look at the last commit, this line is no longer present.